airflow/providers/google/cloud/hooks/bigquery.py

```diff
@@ -240,7 +240,7 @@ def get_pandas_df(
         query. The DbApiHook method must be overridden because Pandas
         doesn't support PEP 249 connections, except for SQLite. See:
 
-        https://github.com/pydata/pandas/blob/master/pandas/io/sql.py#L447
+        https://github.com/pydata/pandas/blob/main/pandas/io/sql.py#L447
```

Comment:
```suggestion
```

I think it could be removed, if we attach link to the branch without specific commit then reference to line(s) could be changes thought the time:

Right not it refers to
```console
        Whether to use nullable dtypes as default when reading data. If
```

Reply:
Good point! I updated the links (they were outdated anyway). I think keeping the links there is still helpful. There is a pandas issue also linked, but reading through that conversation can be cumbersome even though it is good context for _why_ only SQLite is supported with DBAPIv2.